cmd/launcher/desktop.go

```diff
@@ -182,9 +184,10 @@ func runDesktop(_ *multislogger.MultiSlogger, args []string) error {
 		}
 	}()
 
+	// block until a send on showDesktopChan
+	<-showDesktopChan
```

Comment:
Ooh, this is tidy!

Reply:
I was torn between this route and just passing a flag when we spin up desktop. We are able to show the desktop on demand, but as far as I can tell, there is no way to hide the desktop on demand. When you tell systray to shutdown, it exits the program completely. So if we ever want to hide it, we have to kill the process and let a new one spin up hidden.

I chose this route because I think it's rare that we would turn off the desktop and I don't want to be starting / killing processes on a first install where a user is trying auth for the first time.

Reply:
>I think it's rare that we would turn off the desktop

I agree, I think that's a reasonable assumption -- this route makes sense to me

Reply:
I think users will, but I think your reasoning is sound.